libcxx/test/std/localization/locale.categories/category.numeric/locale.num.get/facet.num.get.members/get_long_double.pass.cpp

```diff
@@ -165,10 +166,11 @@ int main(int, char**)
             f.get(cpp17_input_iterator<const char*>(str),
                   cpp17_input_iterator<const char*>(str+sizeof(str)),
                   ios, err, v);
-        assert(base(iter) == str+sizeof(str)-1);
-        assert(err == ios.goodbit);
-        assert(std::isnan(v));
+        assert(base(iter) == str);
+        assert(err == ios.failbit);
+        assert(v == 0.0l);
     }
+#if LDBL_MANT_DIG >= 64
```

Comment:
`1.189731495357231765021264e+49321` does not fit in binary128 too.
What changed that these values no longer work on MSVC?

Reply:
Thanks for the correction. I've verified that `1.189731495357231765021264e+49321` and `1.189731495357231765021264e+49329` consistently result in infinity on MSVC, so perhaps these cases shouldn't be guarded.